js/index.js

```diff
@@ -22,6 +22,57 @@ for (let i = 0; i < skills.length; i++) {
     skill.innerHTML = skills[i];
     skillsList.appendChild(skill);
 }
+// 4.3 Form Basics
+//*******************************************************************************************
+// create messages
+// *******************************************************************************************
+// show the information from the form
+//--------------------------------------------------------------------------------------------
+const messageForm = document.getElementsByName("leave_message");
+console.log(messageForm[0], "Show the data from the form ");
 
+messageForm[0].addEventListener("submit", function(event){
+    event.preventDefault();
 
+const theMesssageForm = event.target;
 
+var name = event.target.name.value;
+var email = event.target.email.value;
+var message = event.target.message.value;
```

Comment:
```suggestion
const name = event.target.name.value;
const email = event.target.email.value;
const message = event.target.message.value;
```
Use `const` by default, or `let` if it will change.